index.html

```diff
@@ -7,92 +7,226 @@
         </div>
       </div>
-    </div>
-    <div class="pros_container">
-      <div class="photo_description_container">
-        <div class="pros_photo"><img class="pros_pic" src="mountains.png" alt="zdjęcie zalet"></div>
-        <div class="paragraph_container">
-          <header class="paragraph_header">Close to nature</header>
-          <p>Every cabin means different areas to explore!</p>
+    </section>
+
+
+    <section class=" fourth_page_container">
+      <div class="flex section_wrapper">
+      <div class="left_side_fourth_page">
+        <div>
+          <img class="girl_pic" src="./girl.png" alt="girl pic">
         </div>
       </div>
-    </div>
-  </div>
-  <div class="third_page_container">
-    <div class="left_side">
-      <div class="some_text">
-        <header class="third_page_header">Work from somewhere special</header>
-        <p>We gather people who love nature and mountains and help them find perfect spots for a few weeks or even
-          months.
-          If your job is fully remote and you daydream of holidays, then why sit at home? Work as you normally do and
-          later let yourself dive into nature. Break the habit.</p>
+      <div class="right_side_fourth_page">
+        <div class="fan_number">
+          <p class="cabin_lovers">
+            +2,000<br>cabin lovers
+          </p>
+        </div>
+        <div class="autor">
+          <p class="fourth_page_footer">
+            Marina Skovic, Visual Artist
+          </p>
+          <ul class="unordered_list_for_numbers">
+            <li class="numbers_fourth_page">
+              <a class="anchor_number" href="#">1</a>
+            </li>
+            <li class="numbers_fourth_page">
+              <a class="anchor_number" href="#">2</a>
+            </li>
+            <li class="numbers_fourth_page">
+              <a class="anchor_number" href="#">3</a>
+            </li>
+          </ul>
+        </div>
       </div>
-      <div class="btn"><button>Book now</button></div>
-    </div>
-    <div class="right_side">
-      <img class="mountains_pic" src="mountain_home.png" alt="widok na góry">
-    </div>
-  </div>
-  <div class="fourth_page_container">
-    <div class="left_side_fourth_page">
-      <div><img class="girl_pic" src="girl.png" alt="zdjęcie dziewczyny"></div>
+      </div>
+    </section>
+
+
+
+    <section class="section fifth_page_container">
+      <div class="flex section_wrapper">
       <div class="newsletter">
-        <label for="newsletter_header"> newsletter comming soon</label>
-        <input type="email" class="email" placeholder="your e-mail address">
+        <div class="input_newsletter">
+          <div class="input_header">
+            <label for="newsletter_header" class="newsletter_info"> newsletter comming soon</label>
+          </div>
+          <div class="input_input">
+            <input type="email" class="email" placeholder="your e-mail address">
```

Comment:
label `for` and input `id` are a pair and should have the same value

Reply:
still not done :)

Reply:
fixed